File: Kattis/laegdyfirlandinu/laegdyfirlandinu.py
import sys

n, m = map(int, input().split())
forecasts = []

for i in range(n):
    forecasts.append(list(map(int, input().split())))

for i in range(1, n - 1):
    for j in range(1, m - 1):
        currentForecast = forecasts[i][j]

        if (forecasts[i-1][j] > currentForecast and
            forecasts[i+1][j] > currentForecast and
            forecasts[i][j-1] > currentForecast and
            forecasts[i][j+1] > currentForecast):
            print("Jebb")
            sys.exit(0)

print("Neibb")

File: Kattis/laegdyfirlandinu/laegdyfirlandinu.cpp
#include <iostream>

int main() {
    int n, m;
    std::cin >> n >> m;
    int forecasts[n][m];

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            std::cin >> forecasts[i][j];
        }
    }

    for (int i = 1; i < n - 1; i++) {
        for (int j = 1; j < m - 1; j++) {
            int currentForecast = forecasts[i][j];

            if (forecasts[i-1][j] > currentForecast &&
                forecasts[i+1][j] > currentForecast &&
                forecasts[i][j-1] > currentForecast &&
                forecasts[i][j+1] > currentForecast
            ) {
                std::cout << "Jebb\n";
                return 0;
            }
        }
    }

    std::cout << "Neibb\n";
    return 0;
}
